### ISSUE TITLE
Select Default Emulator for Systems

### ISSUE THREAD

**Issue by WingofaGriffin** [retrohub-org/retrohub#369]
May be missing this option from my own stupidity, but I would like the option to choose which Emulator opens by default for a certain system.

**Comment by rsubtil** [comment]
Yeah, the system settings is still missing a way to reorder emulators. For now you can remove emulators until your preferred one is indirectly moved to the top, as RetroHub will try emulators from top to bottom. (And you can also select your preferred emulator per game).

I'll implement some reordering functionality soon :+1:
